dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -84,9 +82,6 @@ static LinkageCheckerArguments readCommandLine(String... arguments) throws Parse
       return new LinkageCheckerArguments(parser.parse(options, arguments));
     } catch (IllegalArgumentException ex) {
       throw new ParseException("Invalid URL syntax in Maven repository URL");
-    } catch (ParseException ex) {
-      helpFormatter.printHelp("LinkageChecker", options);
```

Comment:
Does the command still shows available options when invoked without any argument?

Reply:
It does not. We should distinguish help from an error condition. The most common convention seems to be that -h or --help prints help.

Reply:
Common behaviors print the help when a required argument is missing. Would you update this PR to do so?

Reply:
I added a --help/-h option. With that there are no required arguments any more.

Also, I think there's an existing bug where jar files can't be selected.

Reply:
I just ran LinkageCheckerMain without any argument and it existed 0 without showing anything. The empty output does not tell what option I have to add LinkgeCheckerMain.  Would you fix that?

Reply:
> Common behaviors print the help when a required argument is missing.

I'm referring `java` or `javac` command without any argument, for example.

Reply:
I'll see your javac and raise you a gcc:

```
$ gcc
gcc: fatal error: no input files
compilation terminated.
```

but sure, whatever.